.github/workflows/nomad-integration-test.yaml

```diff
@@ -36,7 +37,7 @@ jobs:
         run: sudo apt-get update && sudo apt-get install nomad
       -
         name: Start running nomad in agent mode, then background it
-        run: nomad agent --dev --node dev01 &
+        run: sudo nomad agent --dev --node dev01 &
```

Comment:
Problem is because latest ubuntu is 22.04 which has cgroup v2 enabled by default.

This errors with 

```
Error:     2023-05-05T05:22:59.352Z [ERROR] client.cpuset.v2: failed to ensure nomad parent cgroup exists; disabling cpuset management: error="mkdir /sys/fs/cgroup/nomad.slice: permission denied"
```

However it's recommended to run nomad agents as root even in prod. So we should just do that! 😄 
https://developer.hashicorp.com/nomad/docs/operations/nomad-agent#running-an-agent